articles/postgresql/tutorial-design-database-using-azure-cli.md

```diff
@@ -102,7 +105,7 @@ The result is in JSON format. Make a note of the **administratorLogin** and **fu
 ## Connect to Azure Database for PostgreSQL database using psql
 If your client computer has PostgreSQL installed, you can use a local instance of [psql](https://www.postgresql.org/docs/9.6/static/app-psql.html), or the Azure Cloud Console to connect to an Azure PostgreSQL server. Let's now use the psql command-line utility to connect to the Azure Database for PostgreSQL server.
 
-1. Run the following psql command to connect to an Azure Database for PostgreSQL server
+1. Run the following psql command to connect to an Azure Database for PostgreSQL database:
```

Comment:
In the next paragraph, it says "connects to the default database", so being consistent about where user is connecting to